assets/app/views/directives/_status-icon.html

```diff
@@ -1,11 +1,13 @@
 <span ng-switch="status" class="hide-ng-leave">
+  <span ng-switch-when="Cancelled" class="fa fa-ban text-warning" aria-hidden="true"></span>
   <span ng-switch-when="Complete" class="fa fa-check text-success" aria-hidden="true"></span>
+  <span ng-switch-when="Deployed" class="fa fa-check text-success" aria-hidden="true"></span>
   <span ng-switch-when="Error" class="fa fa-times text-danger" aria-hidden="true"></span>
-  <span ng-switch-when="Cancelled" class="fa fa-ban text-warning" aria-hidden="true"></span>
-  <span ng-switch-when="Succeeded" class="fa fa-check text-success" aria-hidden="true"></span>
   <span ng-switch-when="Failed" class="fa fa-times text-danger" aria-hidden="true"></span>
-  <span ng-switch-when="Terminated" class="fa fa-times text-danger" aria-hidden="true"></span>
+  <span ng-switch-when="New" class="spinner spinner-xs spinner-inline" aria-hidden="true"></span>
   <span ng-switch-when="Pending" class="spinner spinner-xs spinner-inline" aria-hidden="true"></span>
   <span ng-switch-when="Running" class="fa fa-refresh fa-spin" aria-hidden="true"></span>
+  <span ng-switch-when="Succeeded" class="fa fa-check text-success" aria-hidden="true"></span>
+  <span ng-switch-when="Terminated" class="fa fa-times text-danger" aria-hidden="true"></span>
```

Comment:
Will do.  I think `fa-question` looks the best.  `fa-question-circle` is a little hard to see at this size.  Experimented with adding `text-muted` as well on this one, but I think full color is better.

Side by side:
![screen shot 2015-11-20 at 10 45 00 am](https://cloud.githubusercontent.com/assets/280512/11303985/e5156862-8f73-11e5-9b58-c565f003c947.png)
![screen shot 2015-11-20 at 10 46 35 am](https://cloud.githubusercontent.com/assets/280512/11304010/01f76c00-8f74-11e5-8ea3-f55d96814c63.png)